web/styles/popovers.css

```diff
@@ -637,7 +636,7 @@ ul {
 
     .col-left {
         padding: 0 10px 0 0;
-        word-break: break-all;
+        overflow-wrap: break-word;
```

Comment:
During manual testing, I noticed that the profile avatar is being pushed to the right when we use this.

![image](https://github.com/zulip/zulip/assets/82862779/441da375-6a38-42e1-ae76-67b4f73fc7e0)

Reply:
Maybe you can look into `overflow-wrap: anywhere;`, which takes into account the soft wrap opportunities introduced by the word break. I think that might help fix this issue.